---
layout: home
# title: Home
# navigation_weight: 1
---

<style type="text/css">
.image-left {
  display: block;
  margin-left: 20px;
  margin-right: auto;
  float: right;
}
.spaced-lines {line-height: 20pt;} 
</style>

<div class="image-icon-wrapper">
    <img src="images/headshot.jpg" alt="headshot" class="image-left" width="240px" style="border: 5px solid black;">

    <div class="icon-container">
        <a href="https://scholar.google.com/citations?user=tovcdNcAAAAJ" target="_blank">
            <i class="ai ai-google-scholar ai-2x"></i>
        </a>
        <a href="https://orcid.org/0000-0002-0132-3472" target="_blank">
            <i class="ai ai-orcid ai-2x"></i>
        </a>
        <a href="mailto:henri.schmidt@princeton.edu" target="_blank">
            <i class="fa-solid fa-envelope fa-2x"></i>
        </a>
        <a href="https://github.com/schmidt73" target="_blank">
            <i class="fab fa-github-square fa-2x"></i>
        </a>
        <a href="https://x.com/henrismitch" target="_blank">
            <i class="fab fa-twitter-square fa-2x"></i>
        </a>
    </div>
</div>

<div markdown="1" class="spaced-lines">

I am a PhD candidate in Computer Science at [Princeton
University](https://www.cs.princeton.edu). I am very fortunate to be advised by
[Ben Raphael](https://www.cs.princeton.edu/people/profile/braphael). Prior to working with
Ben Raphael, I completed a Bachelor of Science in Math and Computer Science
at [Tufts University](https://www.tufts.edu/), where I worked with 
[Lenore Cowen](https://www.cs.tufts.edu/~cowen/) designing algorithms for 
protein-to-protein interaction networks.

Currently, my research employs techniques from theoretical computer science
and operations research to study problems in computational biology. Specifically, I 
apply these techniques to study phylogenetic optimization problems that arise in 
cancer evolution and lineage tracing. As an example,
my recent work [fastBE](https://doi.org/10.1371/journal.pcbi.1012631) develops 
specialized algorithms to enable large-scale phylogenetic reconstruction 
of tumors using primal-dual methods for solving linear programs.

<h2 style="margin-top:0px;"><i>Recent Updates</i></h2>

<ul>
<li>(January 2025) Our preprint <a href="https://www.biorxiv.org/content/10.1101/2025.01.24.634761v1">Fast tumor phylogeny regression via tree-structured dual dynamic programming</a> is now available on bioRxiv.</li>
<li>(December 2024) The <a href="https://www.biorxiv.org/content/10.1101/2025.02.14.638328v1.abstract">tree labeling polytope</a> manuscript was accepted for presentation at <a href="https://recomb.org/recomb2025/">RECOMB2025</a>.</li>
<li>(December 2024) The fastBE manuscript was published in <a href="https://doi.org/10.1371/journal.pcbi.1012631"> PLoS Computational Biology</a>.</li> 
<li>(September 2024) Attended the ICERM workshop regarding <a href="https://icerm.brown.edu/program/semester_program_workshop/sp-f24-w1#workshopparticipants">Current Methods and Open Problems in Mathematical and Statistical Phylogenetics</a>.</li>
<li>(April 2024) The fastBE paper was presented at RECOMB-CCB.</li>
</ul>

<h2 style="margin-top:0px;"><i>Journal Publications</i></h2>
<p style="margin-top:-15px;">* denotes joint first authorship; author order is rearranged with permission from co-authors. </p>

[<span style="color:#c869bf">Genome-wide CRISPR guide RNA design and specificity analysis with GuideScan2</span>](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-025-03488-8)     
**Henri Schmidt\***, Minsi Zhang\*, Haralambos Mourelatos, Francisco J. Sánchez-Rivera, Scott W. Lowe, Andrea Ventura, Christina S. Leslie, Yuri Pritykin<br>
*Genome Biology,* Volume 26, Article 41, February 26, 2025.

[<span style="color:#c869bf">A regression based approach to phylogenetic reconstruction from multi-sample bulk DNA sequencing of tumors</span>](https://doi.org/10.1371/journal.pcbi.1012631)     
**Henri Schmidt**, Benjamin J. Raphael<br>
*PLoS Computational Biology,* Volume 20, Issue 12, December 2024. [[slides](slides/fastbe.pdf)]

[<span style="color:#c869bf">Multi-center integrated analysis of non-coding CRISPR screens</span>](https://www.nature.com/articles/s41592-024-02216-7)     
David Yao, Josh Tycko, ..., **Henri Schmidt**, ..., Michael C. Bassik, Steven K. Reilly<br>
*Nature Methods,* Volume 21, Issue 4, Pages 723-734, February 2024.

[<span style="color:#c869bf"><i>Startle</i>: A star homoplasy approach for CRISPR-Cas9 Lineage Tracing</span>](https://doi.org/10.1016/j.cels.2023.11.005)     
**Henri Schmidt\***, Palash Sashittal\*, Michelle Chan, Benjamin J. Raphael<br>
*Cell Systems,* Volume 14, Issue 12, December 2023, Pages 1113-1121. [[slides](slides/startle.pdf)]

[<span style="color:#c869bf">A zero agnostic model for copy number evolution in cancer</span>](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1011590)     
**Henri Schmidt**, Palash Sashittal, Benjamin J. Raphael<br>
*PLoS Computational Biology,* Volume 19, Issue 11, November 2023. [[slides](slides/zcnt.pdf)]   

[<span style="color:#c869bf">GLIDER: function prediction from GLIDE-based neighborhoods</span>](https://academic.oup.com/bioinformatics/article/38/13/3395/6586285)     
Kapil Devkota, **Henri Schmidt**, Matt Werenski, James M Murphy, Mert Erden, Victor Arsenescu, Lenore J Cowen<br>
*Bioinformatics,* Volume 38, Issue 13, July 2022, Pages 3395-3406.

[<span style="color:#c869bf">Base editing sensor libraries for high-throughput engineering and functional analysis of cancer-associated single nucleotide variants</span>](https://www.nature.com/articles/s41587-021-01172-3)     
Francisco J Sánchez-Rivera, Bianca J Diaz, Edward R Kastenhuber, **Henri Schmidt**, ..., Scott W. Lowe, Lukas Dow<br>
*Nature biotechnology,* Volume 40, Issue 6, June 2022, Pages 862-873.

## Preprints

[<span style="color:#c869bf">The tree labeling polytope: a unified approach to ancestral reconstruction problems</span>](https://www.biorxiv.org/content/10.1101/2025.02.14.638328v1.abstract)<br>
**Henri Schmidt**, Benjamin J. Raphael<br>
*bioRxiv,* February 19th, 2025. Presenting at RECOMB 2025. [[slides](slides/tlp.pptx)]

[<span style="color:#c869bf">Fast tumor phylogeny regression via tree-structured dual dynamic programming</span>](https://www.biorxiv.org/content/10.1101/2025.01.24.634761v1)<br>
**Henri Schmidt\***, Yuanyuan Qi\*, Benjamin J. Raphael, Mohammed El-Kebir<br>
*bioRxiv,* January 27th, 2025.

[<span style="color:#c869bf">Inferring cell differentiation maps from lineage tracing data</span>](https://doi.org/10.1101/2024.09.09.611835)     
Palash Sashittal, Richard Y. Zhang, Benjamin K. Law, Alexander Strzalkowski, **Henri Schmidt**, Adriano Bolondi, Michelle M. Chan, Benjamin J. Raphael<br>
*bioRxiv,* September 13th, 2024.
&nbsp;
